Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift

```diff
@@ -84,6 +84,8 @@ struct SamplePaywallsList: View {
                     introEligibility: Self.introEligibility
                 )
             )
+        case .customerCenter:
+            CustomerCenterView()
```

Comment:
I believe this is only available in iOS, so we need to wrap all this code into `#if os(iOS)` macros. We can probably add support for other systems soon, but it's a limitation for now.